operator-framework/src/test/java/io/javaoperatorsdk/operator/sample/generickubernetesresource/generickubernetesdependentresourcemanaged/ConfigMapGenericKubernetesDependent.java

```diff
@@ -24,7 +25,7 @@ public class ConfigMapGenericKubernetesDependent extends
   public static final String KEY = "key";
 
   public ConfigMapGenericKubernetesDependent() {
-    super(new GroupVersionKind("", VERSION, KIND));
```

Comment:
The point here was to not use the class, but demonstrate how to do it classless or "dynamically". Would be nicer with a dedicated custom resource for sure. I just used config map. Pls revert this.